applications/nrf5340_audio/tools/buildprog/buildprog.py

```diff
@@ -289,7 +292,15 @@ def __main():
         required=("-M" in sys.argv or "--min_b0n" in sys.argv),
         choices=["external", "internal"],
         default='',
-        help="MCUBOOT with external, internal flash",)
+        help="MCUBOOT with external, internal flash",
+        )
+    parser.add_argument(
+        "--shield",
+        action="store_true",
+        dest="shield",
+        default=False,
+        help="Set when using nRF21540 for extra TX power",
```

Comment:
I think we should be a bit more concise on "FEM" vs "nRF21540"

Reply:
Always use nRF21540_EK